setup.py

```diff
@@ -78,6 +78,7 @@
             "importer = cds_ils.importer.cli:importer",
             "fixtures = cds_ils.cli:fixtures",
             "covers = cds_ils.cli:covers",
+            "patrons = cds_ils.patrons.cli:patrons",
```

Comment:
is there a name conflict when registering this command? since it is the same in app-ils

Reply:
you are right, it was overriding the patrons commands from invenio-app-ils so I renamed it.